markdown-pages/en/tidb/master/releases/release-7.0.0.md

```diff
@@ -0,0 +1,86 @@
+---
+title: TiDB 7.0.0 Release Notes
+---
+
+# TiDB 7.0.0  Release Notes
+
+Release date: xx, 2023
+
+TiDB version: 7.0.0-DMR
+
+Quick access: Quick start | Installation package
+
+In v7.0.0-DMR, the key new features and improvements are as follows:
+
+<table>
+<thead>
+  <tr>
+    <th>Category</th>
+    <th>Feature/Enhancement</th>
+    <th>Description</th>
+  </tr>
+</thead>
+<tbody>
+  <tr>
+    <td rowspan="2">Scalability and Performance<br/></td>
+    <td>Session level SQL plan cache (experimental)</td>
+    <td>Using plan cache can reduce the time for compiling and optimizing SQL queries, which reduces query latency. For each query executed, if a plan doesn't exist, a new one will be stored and re-used next time.</td>
```

Comment:
Automatically re-using plan cache can reduce the time for compiling and optimizing same SQL query patterns, which can reduce query latency without manually set prepare statement in advance.

Reply:
change feature name to 'Session level non-prepared SQL plan cache'